.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,23 @@
+## For issue
+
```

Comment:
(I think we don't need this section if it's in the description section, right?

Reply:
Agree